app/models/maintenance_tasks/progress.rb

```diff
@@ -51,13 +51,13 @@ def text
       count = @run.tick_count
       total = @run.tick_total
       if !total?
-        "Processed #{pluralize(count, 'item')}."
+        "Processed #{pluralize(count, "item")}."
```

Comment:
Technically these could have been in a separate commit before the dependabot upgrade, but it's fine.

Reply:
Yeah, I missed these in #376 😅

Reply:
You didn't miss anything 😄
It's because we merged #377 since then.

Reply:
Rightt, okay I'm off the hook 😂